src/core/public/chrome/ui/header/collapsible_nav.tsx

```diff
@@ -134,6 +136,39 @@ export function CollapsibleNav({
       isDocked={isLocked}
       onClose={closeNav}
     >
+      {customNavLink && (
+        <Fragment>
+          <EuiFlexItem grow={false} style={{ flexShrink: 0 }}>
+            <EuiCollapsibleNavGroup
+              background="light"
+              className="eui-yScroll"
+              style={{ maxHeight: '40vh' }}
+            >
+              <EuiListGroup
+                aria-label={customNavLink.title}
```

Comment:
```suggestion
```

Below we set list `aria-label` attributes to what type of list it is because we have multiple lists in one nav. (So the names are like "Pinned links" and "Recently viewed links".)

With this list having only one item (meaning, it's quick to go through) and being kind of a weird category anyways, I'd rather not name this list. If you think we'll keep using this list for more things, you could try to come up with some sort of category name for it (because "Custom links" doesn't really mean much to users, I think).